### PR TITLE
fixing dependabot build issues

### PR DESCRIPTION
**Pull request by reezarabara-helix**
updated assume_creds to name role session "bot-circleci" if dependabot is triggering builds.

### CODE REVIEW

File: src/commands/test.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ steps:
       name: Act as AWS admin
       command: |
         # Manually assume role to get temporary aws credentials
-        assume_creds=$(aws sts assume-role --role-session-name "$CIRCLE_USERNAME-circleci" --role-arn arn:aws:iam::$AWS_ACCOUNT_ID:role/CiDeployer)
+        assume_creds=$(aws sts assume-role --role-session-name "circleci" --role-arn arn:aws:iam::$AWS_ACCOUNT_ID:role/CiDeployer)
```
Comment (helixliam) [inline src/commands/test.yml:26]: @reezarabara-helix Is there any possibility that conflicts in the role session name will lead to issues when multiple users trigger CircleCI workflows at the same time?

From the [AWS docs](https://docs.aws.amazon.com/cli/latest/reference/sts/assume-role.html#options):
> Use the role session name to uniquely identify a session when the same role is assumed by different principals or for different reasons.

Comment (corrjo) [reply src/commands/test.yml:26]: It won't cause issues, but we can be more precise if needed. we can include one of the circle variables to distinguish which job/repo/branch assumed the role. if thats useful to security. generally we've just used it to tell the difference between machine/human

Comment (helixliam) [reply src/commands/test.yml:26]: Would it make sense to use something like `--role-session-name "${CIRCLE_USERNAME:-bot}-circleci"` to preserve the same level of detail in the audit trail?

Comment (reezarabara-helix) [reply src/commands/test.yml]: Good point. Updated code

**Review by zubinmadon — APPROVED** [review]

**Pull request merged**